index.html

```diff
@@ -2706,6 +2884,54 @@ <h2>
         <h2>
 Metadata Structure
         </h2>
+
+        <p>
+A <a href="metadata-structure">metadata structure</a> is a common and interoperable data structure used to
+communicate information alongside a <a>DID</a> and <a>DID document</a>, and is specifically
+used for the <a>DID resolution</a> function definition in this specification.
+Use of a <a href="metadata-structure">metadata structure</a> is intended to compliment the information available
+within a <a>DID</a> or <a>DID document</a>, and therefore is best suited to carrying
+metadata as an input or output to the function process.
+        </p>
+        
+        <p>
+A <a href="metadata-structure">metadata structure</a> consists of a set of properties.
+Each property has a single name and a single value.
+The name of the property MUST be a single string. 
+Property names MUST be unique within a given set. 
+Each property name MUST map unambiguously to a single value within the set.
+Property names MUST be compared by the exact byte value.
+The value of a property MUST be a single string.
```

Comment:
Properties must be able to be any JSON type - including arrays, booleans, objects, and yes strings.

OAuth metadata contains all of these.  There's no reason to believe that DID metadata won't.